content/develop/interact/search-and-query/administration/configuration.md

```diff
@@ -0,0 +1,582 @@
+---
+aliases: /develop/interact/search-and-query/basic-constructs/configuration-parameters
+categories:
+- docs
+- develop
+- stack
+- oss
+- rs
+- rc
+- oss
+- kubernetes
+- clients
+description: 'Redis Query Engine can be tuned through multiple
+  configuration parameters. Some of these parameters can only be set at load-time,
+  while other parameters can be set either at load-time or at run-time.'
+linkTitle: Configuration parameters
+title: Configuration parameters
+weight: 1
+---
+{{< note >}}
+As of Redis Community Edition 8.0, the `FT.CONFIG SET` and `FT.CONFIG GET` commands are deprecated.
+Configuration parameters for Redis Query Engine are now set in the following ways:
+* At load-time via your `redis.conf` file.
+* At run-time (where applicable) using the [`CONFIG SET`]({{< relref "/commands/config-set" >}}) command.
+{{< /note >}}
+
+## Redis Query Engine configuration parameters
+
+The following table summarizes which configuration parameters can be set at run-time, and compatibility with Redis Software and Redis Cloud.
+
+| Old parameter name                | New parameter name                                              | Run-time             | Redis<br />Software | Redis<br />Cloud |
```

Comment:
Maybe I missed this part, but do we explain what is 'Old' and what is 'New'?
Maybe, instead, use "parameter name (version < 8.0)" and  "parameter name (version 8.0+)"?

Reply:
One more thing we may want to mention - version 8 still supports the "old" names for backward compatibility.

Reply:
Good suggestion! Applied in next push.